index.html

```diff
@@ -0,0 +1,692 @@
+                    <a href="#">Sunday review</a>
+                    <a href="#">Video: Opinion</a>
+                </div>
+                <div class="footer-data">
+                    <h6>ARTS</h6>
+                    <a href="#">Today's Arts</a>
+                    <a href="#">Art & Design</a>
+                    <a href="#">Books</a>
+                    <a href="#">Dance</a>
+                    <a href="#">Movies</a>
+                    <a href="#">Music</a>
+                    <a href="#">Pop culture</a>
+                    <a href="#">Television</a>
+                    <a href="#">Theatre</a>
+                    <a href="#">Video: Arts</a>
+                </div>
+                <div class="footer-data">
+                    <h6>LIVING</h6>
+                    <a href="#">At Home</a>
+                    <a href="#">Automobiles</a>
+                    <a href="#">Games</a>
+                    <a href="#">Education</a>
+                    <a href="#">Food</a>
+                    <a href="#">Health</a>
+                    <a href="#">Jobs</a>
+                    <a href="#">Love</a>
+                    <a href="#">Magazine</a>
+                    <a href="#">Parenting</a>
+                    <a href="#">Real Estate</a>
+                    <a href="#">Recipes</a>
+                    <a href="#">Styles</a>
+                    <a href="#">T Magazine</a>
+                </div>
+                <div class="footer-data">
+                    <h6>MORE</h6>
+                    <a href="#">Reader Center</a>
+                    <a href="#">Wirecutter</a>
+                    <a href="#">Live Events</a>
+                    <a href="#">The learning Network</a>
+                    <a href="#">Tools & Services</a>
+                    <a href="#">Multimedia</a>
+                    <a href="#">Photography</a>
+                    <a href="#">Video</a>
+                    <a href="#">Newsletters</a>
+                    <a href="#">TimesMachines</a>
+                    <a href="#">NYT Store</a>
+                    <a href="#">Times Journeys</a>
+                    <a href="#">Manage My Account</a>
+                </div>
+                <div class="footer-data">
+                    <h6>SUBSCRIBE</h6>
+                    <a class="bold" href="">Home Delivery</a>
+                    <a class="bold" href="">Digital Subscriptions</a>
+                    <a class="bold" href="">Games</a>
+                    <a class="bold" href="">Cooking</a>
+                    <a href="">Email Newsletters</a>
+                    <a href="">Corporate Subscriptions</a>
+                    <a href="">Education rate</a>
+                    <hr class="sub-hr" />
+                    <a href="">Mobile Applications</a>
+                    <a href="">Replica Edition</a>
+                    <a href="">International</a>
+                    <a href="">Canada</a>
+                    <a href="">Espanol</a>
```

Comment:
It seems you forgot to mark the link as a dead link `<a href="#">`

Reply:
Solved